rest/blip_sync.go

```diff
@@ -157,12 +157,12 @@ func (ctx *blipSyncContext) register(profile string, handlerFn func(*blipHandler
 			if response := rq.Response(); response != nil {
 				response.SetError("HTTP", status, msg)
 			}
-			ctx.LogTo("SyncMsg", "#%03d: Type:%s   --> %d %s Time:%v User:%s", handler.serialNumber, profile, status, msg, time.Since(startTime), ctx.effectiveUsername)
+			ctx.LogTo("SyncMsg", "#%03d: Type:%s   --> %d %s Time:%v User:%s", base.MD(handler.serialNumber), base.MD(profile), base.MD(status), base.MD(msg), base.MD(time.Since(startTime)), base.UD(ctx.effectiveUsername))
```

Comment:
I think redacting the duration and status of the requests may impact support too much. I'm unsure of exactly where the line is drawn for anything other than user-data.

@adamcfraser What do you think?

Reply:
@bbrks actually that brings up a question I had -- what's the easiest way for me to preview the post-redacted output?

Reply:
@tleyden Running SG with this in the config should do it:

```json
"logging": {
    "default": {
        "logKeys": ["*"],
        "redactionLevel": "partial"
    }
}
```